Tests/CombineTests/PublisherExtensionsTests.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  PublisherExtensionsTests.swift
+//  SwifterSwift
+//
+//  Created by Heecheon Park on 7/28/23.
+//  Copyright © 2023 SwifterSwift
+//
+
+// swiftlint:disable identifier_name large_tuple
+
+@testable import SwifterSwift
+import XCTest
+
+#if canImport(Combine)
+import Combine
+
+final class PublisherExtensionsTests: XCTestCase {
+
+    func testCombineLatest() throws {
+        if #available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *) {
+            var result: [(Int, Int, Int, Int, Int)] = []
+
+            let p1 = PassthroughSubject<Int, Never>()
+            let p2 = PassthroughSubject<Int, Never>()
+            let p3 = PassthroughSubject<Int, Never>()
+            let p4 = PassthroughSubject<Int, Never>()
+            let p5 = PassthroughSubject<Int, Never>()
+            
+            let cancellable = p1.combineLatest(p2, p3, p4, p5)
+                .sink { result.append(($0, $1, $2, $3, $4)) }
+            
+            defer { cancellable.cancel() }
```

Comment:
I don't think you need to do this. The `Cancellable` should do it automatically when it's deinitialised.

Reply:
handled at c66c7fb4bb52f15ec1c5b34be6c90ef77e380ad5